src/tests/compiler/statement/doStatementTests.ts

```diff
@@ -0,0 +1,26 @@
+import * as ts from "typescript";
+import {expect} from "chai";
+import {DoStatement} from "./../../../compiler";
+import {getInfoFromText} from "./../testHelpers";
+
+function getInfoFromTextWithDoStatement(text: string) {
+    const obj = getInfoFromText(text);
+    const doStatement = (
+        obj.sourceFile.getFirstDescendantByKindOrThrow(ts.SyntaxKind.DoStatement)
+    ) as DoStatement;
+    return {...obj, doStatement};
+}
+
+describe(nameof(DoStatement), () => {
+    const statement = "do {}";
```

Comment:
Wow, it seems so obvious now, but I was just puzzled as to what the expression node was supposed to be. I tried random things like `do (x > 0) {}` because for some reason I thought a `do {} while (x > 0)` would be a combination of a `DoStatement` and a `WhileStatement`.

Reply:
Haha, I've done something similar a few times before too :)